src/App.ts

```diff
@@ -106,43 +107,44 @@ const tokenUsage =
 
 /** App initialization options */
 export interface AppOptions {
-  signingSecret?: HTTPReceiverOptions['signingSecret'];
-  endpoints?: HTTPReceiverOptions['endpoints'];
-  port?: HTTPReceiverOptions['port'];
-  customRoutes?: HTTPReceiverOptions['customRoutes'];
-  processBeforeResponse?: HTTPReceiverOptions['processBeforeResponse'];
-  signatureVerification?: HTTPReceiverOptions['signatureVerification'];
-  clientId?: HTTPReceiverOptions['clientId'];
-  clientSecret?: HTTPReceiverOptions['clientSecret'];
-  stateSecret?: HTTPReceiverOptions['stateSecret']; // required when using default stateStore
-  redirectUri?: HTTPReceiverOptions['redirectUri'];
-  installationStore?: HTTPReceiverOptions['installationStore']; // default MemoryInstallationStore
-  scopes?: HTTPReceiverOptions['scopes'];
-  installerOptions?: HTTPReceiverOptions['installerOptions'];
   agent?: Agent;
-  clientTls?: Pick<SecureContextOptions, 'pfx' | 'key' | 'passphrase' | 'cert' | 'ca'>;
-  convoStore?: ConversationStore | false;
-  token?: AuthorizeResult['botToken']; // either token or authorize
   appToken?: string; // TODO should this be included in AuthorizeResult
+  attachFunctionToken?: boolean;
+  authorize?: Authorize<boolean>; // either token or authorize
   botId?: AuthorizeResult['botId']; // only used when authorize is not defined, shortcut for fetching
   botUserId?: AuthorizeResult['botUserId']; // only used when authorize is not defined, shortcut for fetching
-  authorize?: Authorize<boolean>; // either token or authorize
-  receiver?: Receiver;
-  logger?: Logger;
-  logLevel?: LogLevel;
-  ignoreSelf?: boolean;
+  clientId?: HTTPReceiverOptions['clientId'];
   /**
    * Configurations for the web client used to send Slack API method requests.
    *
    * See {@link https://tools.slack.dev/node-slack-sdk/reference/web-api/interfaces/WebClientOptions} for more information.
    */
   clientOptions?: WebClientOptions;
-  socketMode?: boolean;
-  developerMode?: boolean;
-  tokenVerificationEnabled?: boolean;
+  clientSecret?: HTTPReceiverOptions['clientSecret'];
+  clientTls?: Pick<SecureContextOptions, 'pfx' | 'key' | 'passphrase' | 'cert' | 'ca'>;
+  convoStore?: ConversationStore | false;
+  customRoutes?: HTTPReceiverOptions['customRoutes'];
   deferInitialization?: boolean;
+  developerMode?: boolean;
+  endpoints?: HTTPReceiverOptions['endpoints'];
   extendedErrorHandler?: boolean;
-  attachFunctionToken?: boolean;
+  ignoreSelf?: boolean;
+  installationStore?: HTTPReceiverOptions['installationStore']; // default MemoryInstallationStore
+  installerOptions?: HTTPReceiverOptions['installerOptions'];
+  logger?: Logger;
+  logLevel?: LogLevel;
+  port?: HTTPReceiverOptions['port'];
+  processBeforeResponse?: HTTPReceiverOptions['processBeforeResponse'];
+  receiver?: Receiver;
+  redirectUri?: HTTPReceiverOptions['redirectUri'];
+  scopes?: HTTPReceiverOptions['scopes'];
+  socketModeOptions?: Omit<SocketModeOptions, 'appToken'>;
```

Comment:
@mwbrooks @WilliamBergamin Bundled! 

The caveat here is that `SocketModeOptions` (from the `socket-mode` pkg) requires `appToken`; we derive that from the root level initialization here. I've leveraged `Omit` to appease the type expectation, but open to alternatives (though it seems preferable at first blush to avoid the creation of a near-identical type).